pkg/storage/stores/shipper/compactor/deletion/delete_requests_manager.go

```diff
@@ -163,9 +168,19 @@ func (d *DeleteRequestsManager) Expired(ref retention.ChunkEntry, _ model.Time)
 
 		d.chunkIntervalsToRetain = rebuiltIntervals
 		if len(d.chunkIntervalsToRetain) == 0 {
+			level.Info(util_log.Logger).Log(
+				"msg", "no chunks to retain for delete request",
```

Comment:
This means we delete the whole chunk, right? Would it be good to add that to the log?

Reply:
Good idea. I've added the ChunkID as well.